book/website/community-handbook/fireside-chat/fireside-chat-roles.md

```diff
@@ -0,0 +1,99 @@
+## Co-hosts - Facilitate the Session
+
+Co-hosts are invited to co-design the event with one of *The Turing Way* team members. 
+They enable a discussion that does not look for answers or solutions but creates shared opportunities for members from different communities to explore a topic or challenge from their contexts. 
+If there are solutions available, that is great! But that is not the goal. The hope is that these discussions will stimulate ideas for collaboration beyond the Fireside Chats.
+**We have 2 hosts for each session to allow hosts to also address questions, as well as share tasks such as coordinating with the speakers or monitoring questions.**
+
+* During the event, they will help each other keep track of time, and bring back discussions in the direction that they planned for if the discussion deviates. Support each other in taking audience questions. If due to a technical glitch, one of the hosts/speakers drops, they can continue with the planned agenda by themselves, or ask *The Turing Way* team to step in to help as needed.
+* They get to set the tone and direction of the event and suggest/curate speakers for the panel. The suggested number of the complete panel is 5-6. Please consider diverse perspectives and experiences as well as geographical diversity to make sure we have a balanced discussion.
+* Responsibilities of co-hosts: Supported by *The Turing Way* team where needed
+* Co-hosts will represent their organisation/community at this event - and can choose to include their logo in the event’s promotional materials.
+* Share a short blurb about their organisation and a logo to be included on the Eventbrite page.
+* Decide on the title - this will allow us to put together an Eventbrite page. 
+* Decide on the speakers - they are invited and details are added on the Eventbrite page and other promotional material.
+* If they already know the suggested speaker, we will appreciate it if they can send the invitation and get confirmation using an [email draft](https://docs.google.com/document/d/12VoVexsXXBx25SnAU4mayPw5un-v936g9Ux61DLjGzo/edit).
+* Host 30 mins calls with the speakers before the event to identify what they would like to speak about.
+* Frame questions based on their discussions with the speakers, and invite speakers to help finalise the question (using Slack and the planning document).
+* Considering the flow of the discussion, decide on the order of the questions that will be asked.
+  * Each co-host should have equal opportunity to get the ‘stage time’
+  * They should agree on which questions they will pose to guide the discussion.
+  * They should pose 1 question to each speaker who takes a lead on responding. 1-2 follow-up responses can be invited from other speakers.
+    * Make sure that the speakers know in what order they will be speaking, and what questions will be asked.
+    * Coordinate via the Slack channel, and share any notes about the format via the shared document.
```

Comment:
What happens if the host panels don't use Slack? Should we asked them which communications tool they would like us to use for coordinations?